lib/segment/src/vector_storage/quantized/quantized_scorer_builder.rs

```diff
@@ -172,4 +190,68 @@ impl<'a> QuantizedScorerBuilder<'a> {
             }
         }
     }
+
+    fn new_multi_quantized_scorer<TElement, TMetric, TEncodedQuery>(
+        self,
+        quantized_storage: &'a impl EncodedVectors<TEncodedQuery>,
+    ) -> OperationResult<Box<dyn RawScorer + 'a>>
+    where
+        TElement: PrimitiveVectorElement,
+        TMetric: Metric<TElement> + 'a,
+        TEncodedQuery: 'a,
+    {
+        let Self {
+            quantized_storage: _same_as_quantized_storage_in_args,
+            quantization_config,
+            query,
+            point_deleted,
+            vec_deleted,
+            is_stopped,
+            distance: _,
+            datatype: _,
+        } = self;
+
+        match query {
+            QueryVector::Nearest(vector) => {
+                let query_scorer = QuantizedQueryScorer::<TElement, TMetric, _, _>::new_multi(
+                    vector.try_into()?,
```

Comment:
maybe a nice explicit call to `try_from` to help code navigation

Reply:
fixed